requirements/torch_rocm61.txt

```diff
@@ -0,0 +1,4 @@
+--index-url https://download.pytorch.org/whl/rocm6.1
```

Comment:
Any reason we aren't supplying a rocm62 file?

Reply:
can add rocm6.2 support at a later point.
wanted to ship something, and this PR is large enough as is 😬 

differences in the json keys between rocm-smi for 60 and 61 already made it a bit of a headache to support both

Reply:
Got it, makes sense. Personal view is we should just supply support for N and N-1 ROCm version. So for now 6.2 and 6.1. But we can always update this later on.

Reply:
@eppane is looking into this now

Reply:
Tests seem successful with rocm6.2 so we should be able to provide N and N-1 ROCm version support. 👍